src/utils.js

```diff
@@ -71,11 +71,13 @@ export const pipe = (...fns) => fns.reduce(pipeFns);
 export function getPadding() {
   if (!canUseDOM) return 0;
 
-  const currentPadding = parseInt(document.body.paddingRight, 10) || 0;
-  const clientWidth = document.body ? document.body.clientWidth : 0;
-  const adjustedPadding = window.innerWidth - clientWidth + currentPadding || 0;
+  const existingBodyPaddingRight =
+    parseInt(window.getComputedStyle(document.body).paddingRight, 10) || 0;
+  const scrollBarWidth =
+    window.innerWidth - document.documentElement.clientWidth;
+  const calculatedRightPadding = existingBodyPaddingRight + scrollBarWidth || 0;
```

Comment:
Always returns an `Int`

```suggestion
  const calculatedRightPadding = existingBodyPaddingRight + scrollBarWidth;
```

Reply:
Updated